src/monodroid/jni/android-system.h

```diff
@@ -149,9 +149,14 @@ namespace xamarin { namespace android { namespace internal
 			return application_config.uses_mono_aot;
 		}
 
-		bool is_embedded_dso_mode_enabled () const
+		bool is_embedded_dso_mode_enabled ()
```

Comment:
Don't drop the trailing `const` - it tells the compiler that the function isn't modifying the class state and allows more optimizations in generated code.